src/Altinn.App.Core/Infrastructure/Clients/Register/PersonClient.cs

```diff
@@ -64,16 +63,17 @@ public class PersonClient : IPersonRetriever
         /// <inheritdoc/>
         public async Task<Person?> GetPerson(string nationalIdentityNumber, string lastName, CancellationToken ct)
         {
-            HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Get, $"persons");
-
-            await AddAuthHeaders(request);
+            using(var request = new HttpRequestMessage(HttpMethod.Get, $"persons"))
+            {
```

Comment:
New there is now new syntax for a using declaration that creates less indentation https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-statement
```suggestion
            using var request = new HttpRequestMessage(HttpMethod.Get, $"persons");
```

Just skip the `(` after using and end the line with a `;`

Reply:
Nice! Thanks, great readability improvement!

Reply:
UUps, Should be sure to create a full example when using the suggest feature.